weblate/accounts/pipeline.py

```diff
@@ -379,17 +383,18 @@ def store_email(strategy, backend, user, social, details, **kwargs):
     # The email can be empty for some services
     if details.get("verified_emails"):
         current = set(details["verified_emails"])
-        existing = set(social.verifiedemail_set.values_list("email", flat=True))
+        existing = set(social.verifiedemail_set.values_list("email", "is_deliverable"))
         for remove in existing - current:
-            social.verifiedemail_set.filter(email=remove).delete()
+            social.verifiedemail_set.filter(email=remove[0]).delete()
         for add in current - existing:
-            social.verifiedemail_set.create(email=add)
+            social.verifiedemail_set.create(email=add[0], is_deliverable=add[1])
     elif details.get("email"):
         verified, created = VerifiedEmail.objects.get_or_create(
-            social=social, defaults={"email": details["email"]}
+            social=social, defaults={"email": details["email"], "is_deliverable": details["is_deliverable"]}
         )
-        if not created and verified.email != details["email"]:
+        if not created and verified.email != details["email"] and verified.is_deliverable != details["is_deliverable"]:
```

Comment:
The `is_deliverable` is never set in details so far (2/3):

```suggestion
        if not created and verified.email != details["email"] or not verified.is_deliverable:
```